File: run/lua/gameMain.lua
--[[

  GameMain.lua

  written by ragi.

  2016.06.28

]]

require "gxLib/api"
require "gxLib/visorizer"

g_pFileName = "";

function gameMain()

	Visoriser();

end

function DragAndDrop( pFileName )

	g_pFileName = pFileName;

end

File: src/gameMain.cpp
/*

  GameMain.cpp

  written by ragi.

  2016.06.28
*/
#include <gxLib.h>


//void Visorizer( gxBool bReset = gxFalse );
gxBool GunHoundMain(gxBool bReset);

gxBool GameMain()
{
	//Visorizer();
	//
	//return gxTrue;

	return GunHoundMain(gxFalse);
}

gxBool GameEnd()
{
	return gxTrue;
}

gxBool GameSleep()
{
	return gxTrue;
}

gxBool GameResume()
{
	return gxTrue;
}

gxBool GameReset()
{
	//Visorizer( gxTrue );
	//
	//return gxTrue;

	return GunHoundMain(gxTrue);
}

gxBool DragAndDrop(char* szFileName)
{
	return gxTrue;
}

